js/renovation/ui/responsive_box/__tests__/responsive_box.test.tsx

```diff
@@ -1,20 +1,70 @@
 import { mount } from 'enzyme';
+import each from 'jest-each';
+
 import React from 'react';
 import { ResponsiveBoxProps } from '../responsive_box_props';
 import { ResponsiveBox } from '../responsive_box';
 
+import domAdapter from '../../../../core/dom_adapter';
+import { setWindow } from '../../../../core/utils/window';
+
 it('ResponsiveBox > InitialProps', () => {
   const props = new ResponsiveBoxProps();
   const responsiveBox = mount<ResponsiveBox>(<ResponsiveBox {...props} />);
 
-  expect(responsiveBox.props()).toEqual({});
+  expect(responsiveBox.props()).toEqual({
+    screenByWidth: undefined,
+  });
 });
 
 describe('ResponsiveBox > Attrs', () => {
   describe('cssClasses', () => {
-    it('Check has dx-responsivebox class', () => {
-      const responsiveBox = mount<ResponsiveBox>(<ResponsiveBox />);
-      expect(responsiveBox.getDOMNode().classList.contains('dx-responsivebox')).toEqual(true);
+    each([
+      { clientWidth: 1500, expectedClass: 'dx-responsivebox-screen-lg' },
+      { clientWidth: 1000, expectedClass: 'dx-responsivebox-screen-md' },
+      { clientWidth: 900, expectedClass: 'dx-responsivebox-screen-sm' },
+      { clientWidth: 700, expectedClass: 'dx-responsivebox-screen-xs' },
+    ]).describe('Config: %o', ({ clientWidth, expectedClass }) => {
+      it('Check has valid screen size. Default implementation. Has window', () => {
+        const defaultImplementation = domAdapter.getDocumentElement;
+        domAdapter.getDocumentElement = () => ({ clientWidth });
+
+        const props = new ResponsiveBoxProps();
+        const responsiveBox = mount<ResponsiveBox>(<ResponsiveBox {...props} />);
+        const { classList } = responsiveBox.getDOMNode();
+
+        expect(classList.contains('dx-responsivebox')).toEqual(true);
```

Comment:
Can be tested in a separate test without "each(["
Of course it should be in each config but our testing religion suggests to make a separate test.

Reply:
Yep, fixed :)